admin/workspace-management/process-logging.md

```diff
@@ -65,10 +65,25 @@ To view the process logs from a specific user or workspace, you can use your
 cloud provider's log viewer, or you can use `kubectl` to print the logs:
 
 ```bash
-kubectl logs \
-  -l "com.coder.username=zac" \        # Filter by the user "zac"
-  -l "com.coder.workspace.name=code" \ # Filter by the workspace "code"
-  -c exectrace                         # Only show logs from the sidecar
+kubectl logs /
+--selector="com.coder.username=admin" /         # Filter by the user "admin"
+--selector="com.coder.workspace.name=main" /    # Filter by the workspace "main"
+-c workspace                                    # Only show logs from the workspace
```

Comment:
```suggestion
--container="workspace"                         # Only show logs from the workspace
```

might want to specify the namespace too, which is usually `coder` -- this is the command to show logs for all workspaces that I own:

```
$ kubectl logs --selector="com.coder.username=jawnsy" --container=workspace -n coder
info:    2s elapsed:  33.18% of extraction complete (75 MB/227 MB) at 38 MB/s, estimated time left 4s
{"id":"","environment_id":"","build_id":"","time":"0001-01-01T00:00:00Z","type":"substage","msg":"   2s elapsed:  33.18% of extraction complete (75 MB/227 MB) at 38 MB/s, estimated time left 4s","metadata":null}
info:    4s elapsed:  68.80% of extraction complete (156 MB/227 MB) at 39 MB/s, estimated time left 1s
{"id":"","environment_id":"","build_id":"","time":"0001-01-01T00:00:00Z","type":"substage","msg":"   4s elapsed:  68.80% of extraction complete (156 MB/227 MB) at 39 MB/s, estimated time left 1s","metadata":null}
info:    6s elapsed:  85.28% of extraction complete (193 MB/227 MB) at 32 MB/s, estimated time left 1s
{"id":"","environment_id":"","build_id":"","time":"0001-01-01T00:00:00Z","type":"substage","msg":"   6s elapsed:  85.28% of extraction complete (193 MB/227 MB) at 32 MB/s, estimated time left 1s","metadata":null}
{"id":"","environment_id":"","build_id":"","time":"0001-01-01T00:00:00Z","type":"substage","msg":"Complete","metadata":null}
{"id":"","environment_id":"","build_id":"","time":"0001-01-01T00:00:00Z","type":"stage","msg":"Launching coder agent","metadata":null}
{"ts":"2022-02-25T12:56:33.613843857Z","level":"DEBUG","msg":"successfully spawned cvm!"}
{"id":"","environment_id":"","build_id":"","time":"0001-01-01T00:00:00Z","type":"yield","msg":"","metadata":null}
```

Reply:
@deansheather should look at this, exectrace might have been the correct container name, if we want logs for the exectrace sidecar container instead of the workspace container